src/models/user.ts

```diff
@@ -4,9 +4,10 @@ import { IUser } from 'src/interfaces/user';
 
 const userSchema: Schema = new Schema<IUser>(
   {
-    password: { type: String, required: true },
     email: { type: String, required: true },
+    password: { type: String },
     token: { type: String },
```

Comment:
Please remove these attributes, as it will never be stored in our DB. Remember to update the user interface as well